lighthouse-core/lib/emulation.js

```diff
@@ -63,32 +63,47 @@ const NO_CPU_THROTTLE_METRICS = {
 };
 
 /**
+ *
  * @param {Driver} driver
- * @return {Promise<void>}
+ * @param {LH.Config.Settings} settings
  */
-async function enableNexus5X(driver) {
-  await Promise.all([
-    driver.sendCommand('Emulation.setDeviceMetricsOverride', NEXUS5X_EMULATION_METRICS),
+async function emulate(driver, settings) {
+  let params;
```

Comment:
is this falling back to `any` or is ts smart enough to set this properly without explicit typing?

Reply:
changed up this and got nice typing for it.